yt_robot:
  left_bridge_position_controller:
    type: "position_controllers/JointPositionController"
    joint: base_link2left_bridge
    pid: { p: 40.0, i: 0.0, d: 1.0 }
  right_bridge_position_controller:
    type: "position_controllers/JointPositionController"
    joint: base_link2right_bridge
    pid: { p: 40.0, i: 0.0, d: 1.0 }
  left_back_velocity_controller:
    type: "velocity_controllers/JointVelocityController"
    joint: base_link2left_back_wheel
    pid: { p: 10.0, i: 0.001, d: 1.0 }
  right_back_velocity_controller:
    type: "velocity_controllers/JointVelocityController"
    joint: base_link2right_back_wheel
    pid: { p: 10.0, i: 0.001, d: 1.0 }

